frontend/src/metabase/nav/containers/MainNavbar/MainNavbarContainer/BrowseNavSection.tsx

```diff
@@ -3,6 +3,11 @@ import { c, t } from "ttag";
 import { useUserSetting } from "metabase/common/hooks";
 import CollapseSection from "metabase/components/CollapseSection";
 import CS from "metabase/css/core/index.css";
+import { useSelector } from "metabase/lib/redux";
```

Comment:
Pro tip! Review this file with whitespace hidden.
![image](https://github.com/user-attachments/assets/942f7c7a-4db2-4a2b-8707-9f6ed27ab104)